clang/include/clang/Basic/AttrDocs.td

```diff
@@ -7559,6 +7559,26 @@ The attributes ``__arm_in(S)``, ``__arm_out(S)``, ``__arm_inout(S)`` and
   }];
 }
 
+def ArmAgnosticDocs : Documentation {
+  let Category = DocCatArmSmeAttributes;
+  let Content = [{
+The ``__arm_agnostic`` keyword applies to prototyped function types and
```

Comment:
Probably just me but "prototyped function types" reads a little oddly, I think I'd normally just say "function prototypes".

Reply:
It was written this way to distinguish between "prototyped function types" and (K&R style) "unprototyped function types", where the latter does not support the SME attributes.